assets/js/base/components/block-error-boundary/style.scss

```
@@ -14,17 +14,23 @@
 	margin: 0;
 }
 
+.wc-block-error__image {
+	max-width: 25%;
+}
 .wc-block-error__text,
 .wc-block-error__message {
 	margin: 0;
 }
+.wc-block-error__message {
+	font-style: italic;
+}
 
 @include breakpoint( ">480px" ) {
 	.wc-block-error {
 		flex-direction: row;
 	}
 
 	.wc-block-error__image + .wc-block-error__content {
-		margin-left: $gap;
+		margin-left: $gap-larger;
```

Personally speaking here, but I feel this makes the space too wide. If the previous one was too narrow, what do you think about a middle point (`$gap-large`)?
```suggestion
		margin-left: $gap-large;
```